lib/src/oauth2_flows/implicit.dart

```diff
@@ -53,18 +53,22 @@ class ImplicitFlow {
       timeout.cancel();
       try {
         var gapi = js.context['gapi']['auth'];
-        gapi.callMethod('init', [
-          () {
-            completer.complete();
-          }
-        ]);
+        try {
+          gapi.callMethod('init', [
+            () {
```

Comment:
Couldn't this just be `completer.complete`?

Reply:
I think so... Was trying to avoid changing too much...

Reply:
No worries, it just stuck out and hurt my eyes.